src/components/UpdateVotes.jsx

```diff
@@ -0,0 +1,41 @@
+import { patchVotes } from "../../api";
+import {useState, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
+import '../../styling/Vote.css'
+
+export default function UpdateVotes(){
+    const {article_id} = useParams()
+    const [votesCount, setVotesCount] = useState(0)
+    const [err, setErr] = useState(null)
+
+     const handleVotesClick = () =>{
+        setVotesCount(votesCount+1)
+        patchVotes({article_id}).then(()=>{
```

Comment:
There's no need to put `article_id` in an object when passing it to this function. It doesn't look like it's breaking anything, but just a tiny bit more complicated than it needs to be